go/config/config.go

```diff
@@ -251,6 +252,7 @@ func newConfiguration() *Configuration {
 		DiscoveryMaxConcurrency:                      300,
 		DiscoveryQueueCapacity:                       100000,
 		DiscoveryQueueMaxStatisticsSize:              120,
+		DiscoveryCollectionRetentionSeconds:          120,
```

Comment:
Does this mean `DiscoveryCollectionRetentionSeconds` is ON by default? What is the change in behavior I can expect on my own setup as I deploy this?